v2/datastream-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/spanner/DatastreamToSpannerExceptionClassifierIT.java

```diff
@@ -75,32 +75,6 @@ public static void cleanUp() throws IOException {
     ResourceManagerUtils.cleanResources(spannerResourceManager);
   }
 
-  @Test
-  public void testInterleaveInsertChildBeforeParent() {
```

Comment:
This test should still exist for scenarios where people use interleave in with parent.